src/index.html

```diff
@@ -57,6 +62,16 @@
             </div>
         </div>
     </div>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="http://cdn.jsdelivr.net/jquery.mcustomscrollbar/3.0.6/jquery.mCustomScrollbar.concat.min.js"></script>
+    <script>
+        (function($){
+            $(window).load(function(){
+                $('.searchScroll').mCustomScrollbar();
+                $('.favScroll').mCustomScrollbar();
+            });
+        })(jQuery);
+    </script>
```

Comment:
Yeah, we need to use jQuery. There was never any hate for jQuery when jQuery is needed. Colin (and people) just warn others not to use jQuerry for whats Angular job.